packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -20,6 +20,15 @@ export enum Weekday {
   Saturday
 }
 
+export interface InlineCalendarProps {
+  /** Component wrapping the calendar month when used inline. Recommended to be 'article'. */
+  wrapperComponent?: keyof JSX.IntrinsicElements;
+  /** Title of the calendar rendered above the inline calendar month. Recommended to be a 'title' component. */
+  title?: React.ReactNode;
+  /** Id of the accessible label of the calendar month. Recommended to map to the title. */
+  ariaLabeledby?: string;
```

Comment:
Just missing a lowercase `l` (though curiously the attribute still works in Chrome):

```suggestion
  ariaLabelledby?: string;
```